spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+require './lib/vehicle'
+require './lib/vehicle_factory'
+require './lib/dmv_data_service'
+require './lib/facility_factory'
+
+RSpec.describe FacilityFactory do
+
+  let(:oregon) do 
+    {:title=>"Albany DMV Office", :zip_code=>"97321", :website=>"http://www.oregon.gov/ODOT/DMV/pages/offices/albany.aspx", :type=>"DMV Location", :phone_number=>"541-967-2014", :agency=>"Transportation, Department of ", :location_1=>{:latitude=>"44.632897", :longitude=>"-123.077928", :human_address=>"{\"address\": \"2242 Santiam Hwy SE\", \"city\": \"Albany\", \"state\": \"OR\", \"zip\": \"97321\"}"}}
+  end
+  
+  let(:new_york) do 
```

Comment:
I like that you wrote this to mimic the API data so you can test how your methods are working. Nice job.